apps/extension/src/Setup/Ledger/LedgerConnect.tsx

```diff
@@ -80,8 +80,11 @@ export const LedgerConnect: React.FC<Props> = ({
         setCurrentApprovalStep(3);
         const { ak, nsk } = await ledger.getProofGenerationKey(path);
 
-        // SDK wasm init must be called
-        await initWasm();
+        // We only do this to initialize wasm, props don't matter
```

Comment:
Just a note, though I think this is fine: There is an `init` that is directly exported (same with `*/inline`) that doesn't initialize Sdk, but since we don't use the inline imports in extension, you'd have to pass the wasm as an arg :D Maybe that can be improved in the future, but let's keep it like this for now!